ai_chatbots/chatbots.py

```diff
@@ -479,7 +496,14 @@ def get_llm(self, **kwargs) -> BaseChatModel:
 
     async def get_tool_metadata(self) -> str:
         """Return the metadata for the  tool"""
-        return None
+        return json.dumps(
+            {
+                "edx_module_id": self.edx_module_id,
+                "block_siblings": self.block_siblings,
+                "problem": self.problem,
+                "problem_set": self.problem_set,
```

Comment:
I made a guess about what values might be useful here, let me know if anything should be added or removed.